.github/workflows/preview.yml

```diff
@@ -0,0 +1,33 @@
+name: Surge PR Preview
+
+on:
+  workflow_run:
+    workflows: [ "Gatsby Publish" ]
+    types:
+      - completed
+      - closed
+
+jobs:
+  preview:
+    runs-on: ubuntu-20.04
+    if: github.event.workflow_run.event == 'pull_request' && github.event.workflow_run.conclusion == 'success'
```

Comment:
Wonder if we can move this to the `on:` section

Reply:
While checking if we could, I spotted that the `closed` part isn't right. The closed is a property of the workflow itself, not the workflow_run event. We're only allowed [requested, completed, or in_progress](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run). So I'll fix that. 

I also see that the workflow name is totally wrong. (Sigh). Perils of copying and pasting.

Reply:
However, reading the docs further, it looks to me like 

- we can't move the conclusion `if` to the `on`. https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-a-workflow-based-on-the-conclusion-of-another-workflow seems to be very similar to this, and they use an 'if' to check the conclusion
- we *could* run a branch check in the 'on', except that here we don't do a branch check :(